CorsixTH/Lua/app.lua

```diff
@@ -1576,20 +1590,27 @@ function App:afterLoad()
     self.world.original_savegame_version = old
   end
   local first = self.world.original_savegame_version
+
+  -- Generate the human-readable version number (old [loaded save], new [game], first [original])
```

Comment:
'game' is a bit generic, maybe 'program' ?

Reply:
Or "App" these days :)